docs/_data/nav.json5

```diff
@@ -33,7 +23,9 @@
             { label: 'Breadcrumbs', path: 'breadcrumbs' },
             { label: 'Buttons', path: 'buttons' },
             { label: 'Checkboxes', path: 'checkboxes' },
+            //{ label: 'Colors', path: 'colors' },
```

Comment:
Why not just delete this? Looks like it was added by mistake.

Reply:
It's on purpose. Sort of a TODO